src/main/java/org/jitsi_modified/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -200,7 +200,7 @@ public void reset()
      * @param sendTimeMs the send time of the packet in millis
      * @param payloadSize the payload size of the packet.
      */
-    public void incomingPacketInfo(
+    public synchronized void incomingPacketInfo(
```

Comment:
How closely did you look at the places where synchronized was missing?  I always get nervous when adding a bunch of these based on static analysis.

Reply:
I looked at this one carefully. If you look at the diff below you'll see there used to be a `synchronized` a block that included most of the body of the method. I extended it to the whole method, and I don't see anything suspicious in the lines that are affected.